### DISCUSSION TITLE
Send a session to KotlinConf

### DISCUSSION THREAD

**Discussion by pakoito** [libktx/ktx#140] [Ideas]
I thought about sending a session about writing games with libgdx and libktx myself, and I ended up disregarding it. You should do it if you haven't already. The CFP closes in 14 hours, so hurry up :D 

https://sessionize.com/kotlinconf

**Comment by pakoito** [comment]
I also failed to find any way of contacting you on like twitter or something :P

**Comment by czyzby** [comment]
Not sure if I'm good enough to present on a bigger conference yet - local meetups feel stressful enough, to be honest. :) Maybe next year. Anyway, gamedev is just a hobby for me - it would be much more interesting to hear a full-time game developer presentation on Kotlin in the industry. 

> I also failed to find any way of contacting you on like twitter or something :P

I'm on Kotlin Slack. And I actively respond to issues in my projects. ;)
